probe.h

```diff
@@ -25,12 +25,13 @@
 #define PROBE_OFF     0 // No probing. (Must be zero.)
 #define PROBE_ACTIVE  1 // Actively watching the input pin.
 
-
 // Probe pin initialization routine.
 void probe_init();
 
 // Returns probe pin state. Triggered = true. Called by gcode parser and probe state monitor.
-#define probe_get_state() (!(PROBE_PIN & PROBE_MASK))
+//#define probe_get_state() (!(PROBE_PIN & PROBE_MASK))
+#define probe_get_state() (!(MAGAZINE_ALIGNMENT_PIN & MAGAZINE_ALIGNMENT_MASK))
```

Comment:
Makes me nervous to leave commented out code laying around (and to have changed the values in this macro...).

Might be better to simply alias PROBE_PIN and PROBE_MASK to their magazine equivalents with another macro and leave them as PROBE_PIN/PROBE_MASK in this macro (this will also make it easier to adjust these values later should we re-purpose this functionality)